examples/02_decoding/plot_haxby_searchlight.py

```diff
@@ -127,9 +133,9 @@
 # Because scores are not a zero-center test statistics, we cannot use
 # plot_stat_map
 plot_img(
-    searchlight_img,
+    scores_img,
```

Comment:
Then `searchlight_img` should not be created, right ?

Reply:
Yeah.